src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -32,90 +32,103 @@ trait ResourceRoutes extends UserInfoDirectives with SecurityDirectives with Sam
     }
 
   def resourceRoutes: server.Route =
-    (pathPrefix("config" / "v1" / "resourceTypes") | pathPrefix("resourceTypes")) {
+    pathPrefix("initializeResourceTypes") {
       requireUserInfo { userInfo =>
-        pathEndOrSingleSlash {
-          get {
-            complete {
-              resourceService.getResourceTypes().map(typeMap => StatusCodes.OK -> typeMap.values.toSet)
+        asWorkbenchAdmin(userInfo) {
+          pathEndOrSingleSlash {
+            put {
+              complete {
+                resourceService.initResourceTypes()
+              }
             }
           }
```

Comment:
Everything below is just whitespace changes. Hit "Hide whitespace changes" to make it less messy